contrib/startback-jobs/lib/startback/jobs/support/job_result/embedded.rb

```diff
@@ -5,11 +5,19 @@ class JobResult
         class Embedded < JobResult
 
           def api_serve(api)
-            [
-              200,
-              {"Content-Type" => "application/json"},
-              [job.opResult.to_json]
-            ]
+            unless job.failed?
+              [
+                200,
+                {"Content-Type" => "application/json"},
+                [job.opResult.to_json]
+              ]
+            else
+              [
+                400,
+                {"Content-Type" => "application/json"},
+                [{ error: job.opError }.to_json]
+              ]
+            end
```

Comment:
I'm not sure, actually. A 400 means "the request failed because of the requesting user". Here the request is perfectly valid, the job has failed (which is something completely different)